### PR TITLE
docs: update the steps of Fork and Clone

### PR DESCRIPTION
**Pull request by edutrul**
## Changes

Modified in the Fork and Clone steps, so that it is understood how to solve the GitHub token access problem.

## Documentation (please check one with an [x])

- [ ] I have updated the documentation so that the steps in Fork and Clone can be better understood

### CODE REVIEW

**Review by viceice — COMMENTED** [review]
i think we should link to the getting started page instead. as you can also use ENV to configure the token.

**Comment by rarkins** [comment]
The token part is already covered about 30 lines later (it advises env or CLI)

**Comment by edutrul** [comment]
Thank you for your comments.

Yeah, it is covered 30 lines later, but as a first committer to this project, I am following step by step from top to bottom, and that was a stopper for me. Wondering if could move these 30 lines before so that when following the tutorial, it can be easy to follow instructions.

File: docs/development/local-development.md [hunk 1]
```diff
@@ -91,6 +91,14 @@ To ensure everything is working properly on your end, you must:
...
 1. Verify all tests pass and have 100% test coverage, by running `yarn test`
 1. Verify the installation by running `yarn start`. You must see this error: `You must configure a GitHub personal access token`
+  To solve this problem, you must create a file called config.js and place:
+  
+  ```
+     module.exports = {
+     token: 'xxxxxxxx',
+     };
+  ```
+  Your personal GitHub token must be placed.
```
Comment (rarkins) [inline docs/development/local-development.md:101]:
```suggestion

Do not worry about the credentials failure for now, you will be given instructions on how to configure the token a little later down in this document.

```

**Review by rarkins — APPROVED** [review]

**Pull request merged**